mlos_bench/mlos_bench/tests/services/local/local_exec_test.py

```diff
@@ -85,6 +128,16 @@ def test_run_script_read_csv(local_exec_service: LocalExecService) -> None:
         assert stdout.strip() == ""
         assert stderr.strip() == ""
 
+        if sys.platform == 'win32':
+            # On Windows we need to remove the trailing ' ' from the CSV file
+            # that was written due to the way the Python recomposes shell
+            # expansions by reintroducing ' 's between lists of arguments.
+            with open(path_join(temp_dir, "output.csv"), "rt", encoding="utf-8") as fh_output:
+                lines = fh_output.readlines()
+            with open(path_join(temp_dir, "output.csv"), "wt", encoding="utf-8") as fh_output:
+                for line in lines:
+                    fh_output.write(line.rstrip() + os.linesep)
```

Comment:
Not super pleased about this special case, but I don't see a great way around it.
Inside the subprocess module, Python is taking our list of arguments and inserting a space between them which `cmd` is too dumb to strip back out when doing the redirect.